Configuration management. Basic DevOps tools. Introduction to Ansible

Durf1985 opened the pull request:
# Configuring Ansible and executing simple commands and scripts in GCP

## In the process done:

* Install Ansible and Python environment
* Created config file from Ansible
* Created several different types of inventory files (`yml`, `json` and etc)
* Created simple playbook (`clone.yml`)
* Created simple bash script to generate dynamic `inventory.json`

## How to launch a project:
Run next commands:
* `source /venv/bin/activate`
* `bash create_json.sh`
## How to check the performance:
Run in CLI next command `ansible all -m ping `


## PR checklist

The label with the homework number, in label description - homework lecture and topic

Durf1985 committed:
DO-69 Create deploy script
`deploy.sh` located in `your_repo_directory/`
The files necessary for correct working the script are located
in the directory your_repo_directory/config-scripts/two_vm
script work only with `your_repo_directory/terraform/prod` evnironment

loktionovam reviewed (changes requested):
`app.json`, `db.json` and  `variables.json` now is located in your `config-scripts` directory which is not suitable for them. Place them to `packer` directory and invoke `packer` command with appropriate arguments

Durf1985 committed:
Structural Patch for constant GCP account
* `app.json/db.json/variables.json` moved to the `your_repo/packer`
directory from the `config-scripts` directory.
Modified the script ` config-scripts/two_vm/full_vm_image_with_network`
for new paths to the above files.
Fixed version of terraform 1.3.2.
Created deploy.sh script from KodeKloud playground (one
session from one user accaunt limit -3 hours)

Durf1985 committed:
DO-67 Write simple playbook
Simplification of the inventory.yml file
Deleting a more unnecessary vars.yml file
Creating a playbook in the clone.yml file

loktionovam reviewed (commented):
LGTM

Durf1985 committed:
DO-68 Write a script for dynamic inventory on fly

Instead script use gcp_compute plugin for generate dynamic inventory

Durf1985 committed:
DO-68 v.2 Dynamic inventory with script and json

Changed the location of files. The ones that are necessary
for tests are left
A script has been created for dynamic inventory formation
in the inv file.json addresses that do not exist are specified

loktionovam reviewed (approved):
LGTM. But there would be better if you group your hosts in your script by their labels
